tests/http/bad-actors.test.js

```diff
@@ -1,16 +0,0 @@
-var server = require('../../server');
```

Comment:
I'm happy to re add this if you can give more info about the bug. Right now, there isn't anything in the test about why it's there.

Although the test would need to be re made, as it's not working like it used to (and hasn't for a while)